web-ui/partials/index/panel-config-crush.html

```
@@ -0,0 +1,93 @@
+            不会写入真实密钥；默认用 <code>$PROVIDER_API_KEY</code> 形式占位，你确认无误后再点“保存”。
+        </div>
+        <div class="config-template-hint">
+            参考：<a href="https://github.com/charmbracelet/crush/blob/main/README.md#custom-providers" target="_blank" rel="noopener">Custom Providers</a>
+        </div>
+    </div>
+
+    <div class="selector-section">
+        <div class="selector-header">
+            <span class="selector-title">crush.json</span>
+        </div>
+        <textarea
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)